website/src/components/DashboardHeader/DashboardHeader.test.jsx

```diff
@@ -0,0 +1,25 @@
+import { DashboardHeader } from '..';
+import { ProfileInfo } from '..';
```

Comment:
You can import these as one statement `import { DashboardHeader, ProfileInfo } from '..';`